test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -968,13 +1076,19 @@ private String firstTimeStampFor(int initialTZOffset, int requestTZOffset, Sched
         return activities.stream().map(ScheduledActivity::getGuid).collect(Collectors.toSet());
     }
     
+    // TODO: Make endsOn a constant, no reason for it to be here.
```

Comment:
Please sign and date TODO comments. That way if the TODO is still here in a year, you have a quick reference to when it was added. (Git history can change and isn't a great way to track this) Thanks!